### DISCUSSION TITLE
Como implementar o componente Payment

### DISCUSSION THREAD

**Discussion by renerlemes** [mercadopago/sdk-react#100] [General]
Eu consegui implementar o componente Payment, fiz todas as configurações e cheguei até a parte de mostrar no console depois de clicar no botão as informações do formulário.Porém, fiquei com dúvida do que fazer, eu preciso enviar esses dados para um backend e fazer a transação por lá?

**Comment by lemazzo** [comment]
Oi, Rener! 

Sim, precisa. Compartilho a [documentação de Payment Brick](https://www.mercadopago.com.br/developers/pt/docs/checkout-bricks/payment-brick/default-rendering) que te dará o passo a passo. Provavelmente você já passou pelar etapas de "Configurar o Brick" e "Renderizar o Brick", então deve seguir com "Habilitar pagamento com Mercado Pago" e "Envio de pagamentos". 
Também pode ser útil para você esse [exemplo de integração com Payment Brick](https://github.com/mercadopago/payment-bricks-sample-node) e essa [documentação mais técnica](https://github.com/mercadopago/sdk-js/blob/main/API/bricks/payment.md).

Deve ser suficiente para funcionar bem, mas nos avise se tiver algum problema

**Comment by renerlemes** [comment]
@lemazzo obrigado pela ajuda. Consegui efetuar todas configurações até finalizar o pagamento!
